support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -1652,6 +1655,24 @@ function CheckoutComponent({
 											})}
 										</RadioGroup>
 									</fieldset>
+									<SummaryTsAndCs
+										countryGroupId={countryGroupId}
+										contributionType={
+											productFields.billingPeriod === 'Monthly'
+												? 'MONTHLY'
+												: productFields.billingPeriod === 'Annual'
+												? 'ANNUAL'
+												: 'ONE_OFF'
+										}
+										currency={currencyKey}
+										amount={price}
+										amountIsAboveThreshold={
+											productDescription.label === 'All-access digital' ||
+											productDescription.label === 'Digital + print'
```

Comment:
As these are likely to change, should we use the `productKey`?

Reply:
Ah. yes, better to use that, changed